libs/service-portal/core/src/components/NavigationScreen/Card/WipCard.treat.ts

```diff
@@ -0,0 +1,13 @@
+import { style } from 'treat'
+import { theme } from '@island.is/island-ui/theme'
+
+export const WipBoxSmall = style({
+  background: theme.color.blue100,
```

Comment:
This can all be achieved using the <Box element

`<Box background="blue100" borderRadius="standard" paddingTop={4} />`

Reply:
Style variables should also be in camelCase